kube/client.go

```diff
@@ -226,6 +226,9 @@ func (c *Client) Apply(path, namespace string, dryRun, prune, strict, kustomize
 	} else {
 		cmdStr = strings.Join(args, " ")
 		out, err = kubectlCmd.CombinedOutput()
+		if err != nil {
```

Comment:
still that is the same think 2 times. I was suggesting for the whole:
```
out, err = kubectlCmd.CombinedOutput()
if err != nil {
                 return cmdStr, string(out), err
}
```
to be moved outside the `if/else` block, since it is executed regardless the `cmdStr` content